pkg/kwokctl/runtime/kind/kwok_controller_pod.go

```diff
@@ -25,26 +25,26 @@ import (
 	_ "embed"
 )
 
-//go:generate kubectl kustomize -o kwok_controller_deployment.yaml.tpl .
+//go:generate kubectl kustomize -o kwok_controller_pod.yaml.tpl .
```

Comment:
```suggestion
```

Need remove `kustomization.yaml` from the same directory

Reply:
modified